libs/sdk-ui-filters/src/DateFilter/interfaces/index.ts

```diff
@@ -30,11 +31,15 @@ export interface IUiAbsoluteDateFilterForm extends IAbsoluteDateFilterForm {
     to?: DateString;
 }
 
+export const isUiRelativeDateFilterForm = (obj: unknown): obj is IUiRelativeDateFilterForm => {
```

Comment:
Please add a TSDoc and a release tag `@beta`. Also please make sure this function is exported by the `sdk-ui-filters` package.